polar-core/src/kb.rs

```diff
@@ -45,16 +45,18 @@ impl KnowledgeBase {
         self.id_counter.clone()
     }
 
+    pub fn temp_prefix(nom: &str) -> String {
```

Comment:
Could we maybe just add a comment explaining what this is for?

Reply:
👍